src/platform_impl/windows/mod.rs

```diff
@@ -17,7 +17,9 @@ use crate::window::Theme;
 
 #[derive(Clone)]
 pub struct PlatformSpecificWindowBuilderAttributes {
+    // TODO: parent and menu are mutually exclusive; a child window cannot have a menu!
```

Comment:
This should be added the `with_menu` documentationa and I would say a `warn` during the creation if that case is hit would be enough